R/check.R

```diff
@@ -1,9 +1,9 @@
 #' @export
 check_model_pars.tmb_code <- function(x, fixed, random, derived, drops) {
-  check_string(fixed)
-  checkor(check_null(random), check_vector(random, character(0)))
-  checkor(check_null(derived), check_vector(derived, character(0)))
-  checkor(check_null(drops), check_vector(drops, character(0)))
+  chk_string(fixed)
+  chkor(chk_null(random), chk_vector(random, character(0)))
```

Comment:
this 
```
chkor(chk_null(random), chk_vector(random, character(0)))
```

can be replaced by 
```
chk_null_or(random, chk_vector, character(0))
```

Reply:
the change was made to confint.R but replacing with `chk_null_or(random, chk_vector, character(0))` throws the following error:

```
Error (test-zzzanalyse.R:50:1): analyse
Error: Unused argument (character(0)).
Backtrace:
  1. mbr::model(...) test-zzzanalyse.R:50:0
  2. mbr:::model.character(...)
  4. mbr:::model.mb_code(...)
  5. mbr:::model_mb_code(...)
  7. tmbr::check_model_pars.tmb_code(...)
  8. chk::chk_null_or(random, chk_vector, character(0)) /Users/NadineHussein/Code/tmbr/R/check.R:4:2
  9. chk:::abort_null_or(x, msg) /Users/NadineHussein/Code/chk/R/chk-null-or.R:27:2
 10. chk::abort_chk(msg, x = x) /Users/NadineHussein/Code/chk/R/chk-null-or.R:51:2
 11. chk::err(..., n = n, tidy = tidy, .subclass = "chk_error") /Users/NadineHussein/Code/chk/R/utils.R:18:2
 12. rlang::exec(abort, msg, .subclass = .subclass, !!!args[named]) /Users/NadineHussein/Code/chk/R/err.R:86:2
```

Reply:
My bad - can you replace both with 
```
chk_null_or(random, chk_character)
```